### PR TITLE
PAE-590 - Remove login form

### PR DESCRIPTION
**Pull request by luismorenolopera**
Description
---------------
Remove the login form in the pearson-vue-theme.

Screenshots
-----------------

**Before:**
image

**After:**
image

**Mobile (width=480px):**
image

### CODE REVIEW

**Review by diegomillan — APPROVED** [review]

**Review by ivanvgh — APPROVED** [review]

File: edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss [hunk 1]
```diff
@@ -308,12 +306,28 @@ iframe.rocket-chat-view {
...
+    color: $vue-black;
+  }
 }
+
 .login-register .action-primary {
   width: 100%;
   display: block;
 }
 
+.login-container {
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss:325]: Does this work for mobile views too? @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/templates/student_account/login.underscore [hunk 2]
```diff
@@ -1,70 +1,62 @@
...
-  </div>
+  <div class="signin-container">
+    <div class="js-form-feedback" aria-live="assertive" tabindex="-1"></div>
 
 
-  <h2><%- gettext("Sign in") %></h2>
+    <div class="toggle-form">
+      <h1><%- gettext("Welcome!") %></h1>
+      <br>
+      <span class="text"><%- gettext("Sign in or create an account to access your courseware") %></span>
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/templates/student_account/login.underscore]: Is this the required text? @luismorenolopera Please add a period at the end of the line if so.

Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/templates/student_account/login.underscore:14]: I have left an image as a reference for a 480px screen @Squirrel18

File: edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss (hunk 1 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss:325]: I have left an image as a reference for a 480px screen @Squirrel18

File: edx-platform/pearson-vue-theme/lms/templates/student_account/login.underscore (hunk 2 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/templates/student_account/login.underscore]: yes, is required

Comment (Squirrel18) [reply edx-platform/pearson-vue-theme/lms/templates/student_account/login.underscore]: So plaese add the period at the end of the line.

File: edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss (hunk 1 above)
Comment (Squirrel18) [reply edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss:325]: But, it works well on mobile views, right?

Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss:325]: yes

File: edx-platform/pearson-vue-theme/lms/templates/student_account/login.underscore (hunk 2 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/templates/student_account/login.underscore]: done

**Review by Squirrel18 — APPROVED** [review]

**Pull request merged**